operator/cmd/mesh/operator-remove.go

```diff
@@ -78,13 +78,17 @@ func operatorRemove(cmd *cobra.Command, args *RootArgs, orArgs *operatorRemoveAr
 		l.LogAndFatal(err)
 	}
 
+	switch {
 	// If the user is performing purge but also specified a revision, an error message
 	// should be displayed and do nothing
-	if orArgs.purge && orArgs.revision != "" {
+	case orArgs.purge && orArgs.revision != "":
```

Comment:
I think `purge` takes precedence, regardless of the specified revision. We don't care about/set revision if the operation is a `purge`.

There may be a situation where a user specifies both 'purge' and a 'revision'. In such cases, we may need to warn the user that all revisions will still be purged even if they specify the revision.

Reply:
I did not modify the logic here, this is because of the previous discussion: https://github.com/istio/istio/pull/41548#discussion_r1009826463

Reply:
I see. I think it's fine to provide a warning instead of error, since we still have the confirmation before the actual removal, and users will see that they gonna purge everything. This is more consistent to the uninstall command. @zirain Do you have any ideas?

Reply:
we should keep same with `istioctl uninstall`

Reply:
OK